notebooks/simone_pipeline.md

```diff
@@ -0,0 +1,134 @@
+# Summary of Simone's pipeline
+
+## Overview of methods needed
+
+1. 3d gaussian filter (`skimage.filters.gaussian` or `scipy.ndimage.gaussian`) -> `starfish.filters.gaussian` (already present)
+2. `ndimage.gaussian_laplace` -> `starfish.filters.gaussian_laplace`
+	1. Can we assume that any trimming of the image to fit within the dynamic range should be associated with the function that produces the aberration? E.g. `gaussian_laplace` can produce negative values -- should we wrap that function so that it doesn't have this characteristic?
```

Comment:
I think it's OK for values to go negative -- if that's the intended consequence of the filter. Negative values make it hard to write out pngs and tiffs, but i don't think they seriously break anything further on down the line (e.g., spot detection, decoding, etc.). 

perhaps, as a start, we simply log a warning if we detect negative values in the image after filtering, then think about what to do if that happens. alternatively, we can specify various flags for what to do if we see funky dynamic range issues. e.g., in opencv, there's often a flag called preserve_range=True/False that handles this case. 

i think starting with the warning first, but keeping these other options in mind is a good start.

Reply:
> there's often a flag called preserve_range=True/False that handles this case

Got it, I wasn't sure what that referred to. That's helpful. If you happen to know this already, do you know if it rescales such that min(image) becomes zero, or if it typically clips such that `x[x < 0] = 0`? I'm trying to get a sense of the norms. If you don't know I'll look it up.